CRM/Extension/Container/Basic.php

```diff
@@ -98,7 +98,7 @@ class CRM_Extension_Container_Basic implements CRM_Extension_Container_Interface
   public function __construct($baseDir, $baseUrl, ?CRM_Utils_Cache_Interface $cache = NULL, $cacheKey = NULL, ?int $maxDepth = NULL) {
     $this->cache = $cache;
     $this->cacheKey = $cacheKey;
-    $this->baseDir = rtrim($baseDir, '/');
+    $this->baseDir = rtrim($baseDir, DIRECTORY_SEPARATOR);
```

Comment:
I'm not sure all the codepaths that lead to this point, but I think sometimes paths can use forward slashes on windows too (php uses them interchangeably).
So maybe this ought to be
```suggestion
    $this->baseDir = rtrim($baseDir, '/'. DIRECTORY_SEPARATOR);
```
What do you think @demeritcowboy ?

Reply:
Agreed